src/shared/api/auth/sign-in.ts

```diff
@@ -0,0 +1,32 @@
+import { http } from 'shared/api/axios';
+import { AxiosError } from 'axios';
+
+type TSignIn = {
+  login: string,
+  password: string
+};
+
+type TSuccessResponse = 'Ok';
+
+type TBadResponse = {
+  reason: string
+};
+
+const instanceHttp = http.getInstance();
+
+export const signIn = async ({ login, password }: TSignIn) => {
+  try {
+    const response: TSuccessResponse = await instanceHttp.post('/auth/signin', {
```

Comment:
```suggestion
export const signIn = async ({ login, password }: TSignIn) => {
  try {
    const response: TSuccessResponse = await http.getInstance().post('/auth/signin', {
```
Можно прям проще описать)

Reply:
`TSuccessResponse` будет более правильно прокинуть в `post` через дженерик, тогда тип выведется автоматически